src/components/address/Address.jsx

```diff
@@ -113,66 +121,64 @@ const Address = ({
         <SelectField
           placeholder="Select a country"
           className="SelectField Field--Country"
-          value={address.country_code}
+          value={address?.country_code ?? ''}
           messageType={errors && errors?.country && 'alert' || ''}
           messageText={errors && errors?.country && 'Select a country' || ''}
-          onChange={(e) => dispatch({
-            type: 'country_code',
-            payload: e.target.value,
-          })}
-          onBlur={submit}
+          onChange={(e) => {
+            onChange({
+              country_code: e.target.value,
+            });
+          }}
```

Comment:
Insanely nit picky haha but is there a reason why the callback for this onChange is formatted differently from the others?

Reply:
This is a fair callout. It is inconsistent in a few places. I've updated this